### ISSUE TITLE
End-to-end alpha test

### ISSUE THREAD

**Issue by hlydecker** [Sydney-Informatics-Hub/aerial-segmentation#36]
## Context
Let's test basic functionality of all three software streams and the core feature set.

## Tasks
- [ ] download aerial imagery from OSM or sixmaps for a geospatial extent that is the size of four maximum zoom (level 21) tiles. Download this 2x2 grid of tiles and merge into one geotiff.
- [ ] Run through the gis2coco conversion utility to cut into tiles and convert into pings for running through prediction
- [ ] Run through prediction script using the proof of concept model weights and configs for buildings
- [ ] Run through prediction script using the proof of concept model weights and configs for trees
- [ ] Convert predictions for buildings and trees into geojson
- [ ] Render the results

Once all of this is done, and any gaps in code are identified and solved, we can then create a gradio, streamlit, or even a colab that showcases this full functionality.
